buildSrc/subprojects/plugins/src/main/kotlin/org/gradle/gradlebuild/unittestandcompile/UnitTestAndCompilePlugin.kt

```diff
@@ -163,7 +165,7 @@ class UnitTestAndCompilePlugin : Plugin<Project> {
                 jvmArgs("--add-opens", "java.base/java.util=ALL-UNNAMED")
             }
             // Includes JVM vendor and major version
```

Comment:
This comment is no longer true, let's remove it. The property name should also be changed.

I guess we already track the version automatically in the JavaCompile task?

Reply:
The String actually contains vendor and the major version. I renamed the property on `JavaInstallation` to `vendorAndMajorVersion`.

The major Java version is automatically tracked by the compilation and test tasks, we still re-add it as an input again, since there is currently no easy way to obtain it from `JavaInstallationProbe`: https://github.com/gradle/gradle/blob/228ae555db300e2b9fd6cb9755bc1918201e2cab/subprojects/platform-jvm/src/main/java/org/gradle/jvm/toolchain/internal/JavaInstallationProbe.java#L101-L109